python/ray/_raylet.pyx

```diff
@@ -3288,7 +3298,7 @@ def check_health(address: str, timeout=2, skip_version_check=False):
             check_status(PythonCheckGcsHealth(
                 c_gcs_address, c_gcs_port, timeout_ms, c_ray_version,
                 c_skip_version_check, c_is_healthy))
-    except RpcError:
+    except (RpcError, GetTimeoutError):
```

Comment:
`GetTimeoutError` is for `ray.get()`?

I think `RpcError(rpc_code=DEADLINE_EXCEEDED)` is the one we want to raise